Utils/Dataflow/pyDKB/common/hdfs.py

```diff
@@ -194,21 +246,44 @@ def listdir(dirname, mode='a'):
 
 
 def basename(path):
-    """ Return file name without path. """
+    """ Return file name without path.
+    
+    :param path: path to a file
+    :type path: string
```

Comment:
If `path` is `None`, it is taken as empty string -- I believe it should be mentioned somewhere...